pages/_/marketing-candidate/download.js

```diff
@@ -16,83 +16,60 @@ const STYLES_ROOT = css`
 `;
 
 const STYLES_H1 = css`
-  font-size: 1.953rem;
-  line-height: 1.25;
-  padding: 0px 0px 32px 0px;
+  font-size: ${Constants.typescale.lvl5};
+  line-height: 1.3;
+  padding: 0px 0px 24px 0px;
+  letter-spacing: -0.021rem;
   width: 100%;
-  color: ${Constants.system.pitchBlack};
-  @media (max-width: ${Constants.sizes.mobile}px) {
-    font-size: 1.953rem;
+  color: ${Constants.system.slate};
+  font-weight: 800;
+
+  @media (max-width: ${Constants.sizes.tablet}px) {
+    font-size: ${Constants.typescale.lvl5};
     padding: 0px 0px 16px 0px;
-    line-height: 1.25;
   }
-`;
 
-const STYLES_H2 = css`
-  font-size: 1.25rem;
-  line-height: 1.25;
-  padding: 0px 0px 32px 0px;
-  width: 100%;
-  color: ${Constants.system.darkGray};
   @media (max-width: ${Constants.sizes.mobile}px) {
-    font-size: 1.25rem;
-    padding: 0px 0px 8px 0px;
-    line-height: 1.5;
+    font-size: ${Constants.typescale.lvl3};
   }
 `;
 
 const STYLES_H3 = css`
-  font-size: 1.563em;
+  font-size: ${Constants.typescale.lvl2};
   line-height: 1.5;
-  padding: 0px 0px 16px 0px;
-  color: ${Constants.system.darkGray};
-  @media (max-width: ${Constants.sizes.mobile}px) {
-    font-size: 1rem;
-    padding: 0px 0px 8px 0px;
-    line-height: 1.5;
-    color: ${Constants.system.moonstone};
+  letter-spacing: -0.014rem;
+  padding: 0 0 24px 0;
+  color: ${Constants.system.slate};
+
+  @media (max-width: ${Constants.sizes.tablet}px) {
+    font-size: ${Constants.typescale.lvl2};
   }
-`;
 
-const STYLES_P = css`
-  font-size: 1rem;
-  color: ${Constants.system.black};
   @media (max-width: ${Constants.sizes.mobile}px) {
-    font-size: 0.78rem;
+    font-size: ${Constants.typescale.lvl1};
   }
 `;
 
-const STYLES_A = css`
-  :link {
-    color: ${Constants.system.darkGray};
-    background-color: transparent;
-    text-decoration: none;
-  }
-  :hover {
-    color: ${Constants.system.pitchBlack};
-    background-color: transparent;
-    text-decoration: none;
-  }
-  :active {
-    color: yellow;
-    background-color: transparent;
-    text-decoration: none;
-  }
+const STYLES_TEXT_BLOCK = css`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 56vw;
```

Comment:
try sticking to this strategy:

```js
max-width: someValuepx
width: 100%;
```

I think `56vw` provides inconsistent results.